static/fonts/classic-console/font.css

```diff
@@ -0,0 +1,4 @@
+@font-face {
+  font-family: "Classic Console";
+  src: local('*'), url(clacon.woff) format('woff'), url(clacon.ttf) format('truetype');
```

Comment:
Can you please include woff2 as it's more optimized, and remove woff & ttf?

Reply:
Done!